lib/demo_mode/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module DemoMode
-  VERSION = '2.3.0'
+  VERSION = '2.4.0'
```

Comment:
This should probably be v3.0.

If someone upgrades to this version of the gem, it won't work unless they run a migration to add the status column. I'd call that a breaking change.